Description: Remove non-ASCII characters from dbus.cabal
  Haddock run in an sbuild fails if the Cabal description contains non-ASCII
  characters because it runs with LANG=C.
Origin: upstream, commit:jmillikin@gmail.com-20120623214536-chj7g8fifz99xezo
--- a/dbus.cabal
+++ b/dbus.cabal
@@ -13,15 +13,15 @@

 synopsis: A client library for the D-Bus IPC system.
 description:
-  D‑Bus is a simple, message-based protocol for inter-process
+  D-Bus is a simple, message-based protocol for inter-process
   communication, which allows applications to interact with other parts of
   the machine and the user's session using remote procedure calls.
   .
-  D‑Bus is a essential part of the modern Linux desktop, where it replaces
+  D-Bus is a essential part of the modern Linux desktop, where it replaces
   earlier protocols such as CORBA and DCOP.
   .
-  This library is an implementation of the D‑Bus protocol in Haskell. It
-  can be used to add D‑Bus support to Haskell applications, without the
+  This library is an implementation of the D-Bus protocol in Haskell. It
+  can be used to add D-Bus support to Haskell applications, without the
   awkward interfaces common to foreign bindings.
   .
   Example: connect to the session bus, and get a list of active names.
